README.md

```diff
@@ -3,35 +3,82 @@
 ## Overview
 The edX learning platform's frontend for enterprise learners.
 
-## Getting started
+## Setup for development
 
-- Clone this repository locally with `git clone <repo URL>`
+Install nodejs (using nvm maybe a good idea to be able to switch node versions and because there is an .nvmrc file that will be usable by nvm)
 
-##### How to set up the dev environment
 To run the project, first install dependencies
 
-`$ npm i`
+`$ nvm use` # will install/setup node needed for this project
+`$ npm i`   # will install deps
 
 then, start the dev server:
 
 `$ npm start`
 
-### Where is it running?
-The project itself will run on http://localhost:8734
+Visit http://localhost:8734
 
-### Setup prerequisites for full development
+At this point, it will ask you to login. Use the user indicated in devstack (edx@example.com, edx). When logged in, expect to see a 404 since we are not setup
```

Comment:
odd. I did not get a 404 here. I think I had setup devstack and logged in to check the learner portal, but I still wouldn't have setup any additional information. 🤷

Reply:
You'd see a 404 page at the root URL of the service. When on the route for a specific enterprise (localhost:8734/test-enterprise/), you may see a 404 if that Enterprise Customer does not have the "Enable learner portal" flag set (see the `EnterpriseCustomer` model in edx-enterprise).